interoperator/pkg/internal/resources/resources.go

```diff
@@ -491,26 +492,33 @@ func (r resourceManager) deleteSubResource(client kubernetes.Client, resource *u
 				Name:      resource.GetName(),
 				Namespace: resource.GetNamespace(),
 			}
-			err := client.Get(context.TODO(), namespacedName, resource)
-			if err != nil {
-				return err
-			}
-			content := resource.UnstructuredContent()
-			statusInt, ok := content["status"]
-			var status map[string]interface{}
-			if ok {
-				status, ok = statusInt.(map[string]interface{})
-				if !ok {
-					return fmt.Errorf("status field not map for resource %v", resource)
+
+			err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+				err := client.Get(context.TODO(), namespacedName, resource)
```

Comment:
Will it be a good idea to ignore `IsNotFound ` errors and not retry in that case? (It will retry [10 times](https://github.com/kubernetes/client-go/blob/master/util/retry/util.go#L28))

Reply:
IMO it will end up block sync operations like docker, bindings

Reply:
Retry happens only on Conflict error